xcode-build.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+function get_libs {
+    echo "============= Getting Libs ============="
+
+    curl -O https://www.libsdl.org/release/SDL2-2.0.9.zip
```

Comment:
This might be a nitpick, but the tar.gz would be faster since the size is smaller (true for all zip listed here)

Reply:
It is the habit to use zip in MacOS 😅 we'll update it